### PR TITLE
docs: fix promtail docs links in loki installation page

### PR DESCRIPTION
**Pull request by mustafacansevinc**
**What this PR does / why we need it**: fixes link to "syslog-reciever" and "systemd-journal" config docs

**Which issue(s) this PR fixes**: -

**Special notes for your reviewer**: -

**Checklist**
- [x] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: docs/sources/installation/helm.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ spec:
 
 In order to receive and process syslog messages in Promtail, the following changes will be necessary:
 
-* Review the [Promtail syslog-receiver configuration documentation](/docs/clients/promtail/scraping.md#syslog-receiver)
+* Review the [Promtail syslog-receiver configuration documentation](../../clients/promtail/scraping/#syslog-receiver)
```
Comment (kavirajk) [inline docs/sources/installation/helm.md:132]:
```suggestion
* Review the [Promtail syslog-receiver configuration documentation](../../clients/promtail/scraping.md#syslog-receiver)
```
Why extra `/` at the end? Still the filename is same `scraping.md` correct?

File: docs/sources/installation/helm.md [hunk 2]
```diff
@@ -153,7 +153,7 @@ syslogService:
 
 In order to receive and process syslog message into Promtail, the following changes will be necessary:
 
-* Review the [Promtail systemd-journal configuration documentation](/docs/clients/promtail/scraping.md#journal-scraping-linux-only)
+* Review the [Promtail systemd-journal configuration documentation](../../clients/promtail/scraping/#journal-scraping-linux-only)
```
Comment (kavirajk) [inline docs/sources/installation/helm.md:156]:
```suggestion
* Review the [Promtail systemd-journal configuration documentation](../../clients/promtail/scraping.md#journal-scraping-linux-only)
```

Same here!. Filename is still `scraping.md` correct?

**Review by kavirajk — COMMENTED** [review]
@mustafacansevinc Thanks for the fix :).

Added few minor suggestions!

File: docs/sources/installation/helm.md (hunk 1 above)
Comment (mustafacansevinc) [reply docs/sources/installation/helm.md:132]: Right, filename is still same. But the url doesn't work with the .md extension as you can test from https://grafana.com/docs/loki/latest/clients/promtail/scraping.md (404) https://grafana.com/docs/loki/latest/clients/promtail/scraping (200)

You can click on https://grafana.com/docs/loki/latest/clients/promtail/scraping#syslog-receiver and see that the last `/` is added to the url automatically before the `#`.

Comment (kavirajk) [reply docs/sources/installation/helm.md:132]: Cool. Nice! thanks for clarification! :+1:

**Review by kavirajk — APPROVED** [review]
LGTM! :+1:

**Review by owen-d — APPROVED** [review]

**Pull request merged**